### PR TITLE
Switch to SLE12 for Cloud 6.x

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by jdsn** [comment]
:+1:  lgtm, anyhow just retriggered a self-gating build with develcloud6

**Comment by dirkmueller** [comment]
jenkins is green, so lets merge it.

File: scripts/mkcloud [hunk 1]
```diff
@@ -1179,8 +1176,10 @@ function sanity_checks()
         if the machine has a global public IP
         see bnc#928384 and CVE-2012-3411"
 
-    if [[ $want_sles12 ]] && [[ $want_sles12_controller ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 4 ]] ; then
-        complain 113 "Please increase number of nodes for this setup, minimal nodenumber=4"
+    if ! iscloudver 6plus; then
```
Comment (mjura) [inline scripts/mkcloud]: In mkcloud script there is missing function `iscloudver`

```
scripts/mkcloud: line 1179: iscloudver: command not found
```

Comment (mjura) [inline scripts/mkcloud]: If we want to go with full Controller node in SUSE Cloud 6, then we can remove whole this condition

**Pull request merged**